misc updates to workshop host information

danielle06 opened the pull request:
Updated verbiage so that Centrally and Self-Organised workshop references link to their definitions in the respective handbook (links are not live yet, so I will need to go back and add once they are). 

Deleted information about guidelines - we will not have requirements included in the FAQ. Will link out to that instead. 

Rearranged some information for ease of reading with all of the changes.

Please delete this line and the text below before submitting your contribution.

danielle06 committed:
Update host-workshop.md

Updated verbiage so that Centrally and Self-Organised workshop references link to their definitions in the respective handbook (links are not live yet, so I will need to go back and add once they are). 

Deleted information about guidelines - we will not have requirements included in the FAQ. Will link out to that instead. 

Rearranged some information for ease of reading with all of the changes.

maneesha committed:
use relative urls & site params

maneesha commented:
@danielle06 Is this covered by the updates you put in last week?  If so, where should we link to where you have left the links blank?

danielle06 commented:
@maneesha , thanks for checking back on this. Centrally-Organised will link to https://carpentries.org/workshops/workshops-faq/#centrally-organised-workshops and Self-Organised will link to the Self-Organised section in the [Instructor Handbook FAQ](https://docs.carpentries.org/handbooks/instructors.html#faq) - I don't see this as live yet

maneesha commented:
@danielle06 Can you remind me of which PR updated the the Self-Organised section in the Instructor Handbook FAQ?  I know you put in a lot of changes recently, so maybe one of them did not go through when I merged everything?

danielle06 commented:
sorry, @maneesha - I had to find it! Line 439 https://github.com/carpentries/docs.carpentries.org/commit/5589d2e3eebccd416a137ed80f321037defdf7dc

maneesha commented:
@danielle06 I think some things may have gotten mixed up with all the changes we have been making.  In this current PR, I see two places where links need to be filled in but they are both about SOW, not COW. Can you clarify please?

danielle06 commented:
@maneesha , I apologise for the confusion! Looking at line 16 in this current PR: 

1. A [Self-Organised workshop] should link to the Self-Organised section I added in the Instructor handbook (**Line 441** https://github.com/carpentries/docs.carpentries.org/commit/5589d2e3eebccd416a137ed80f321037defdf7dc). 

2. Please visit the [Instructor Handbook FAQ] should link to the Self-Organised section I added in the Instructor handbook (**Line 452** https://github.com/carpentries/docs.carpentries.org/commit/5589d2e3eebccd416a137ed80f321037defdf7dc)

I don't know what I was looking at when I mentioned the COW reference. I hope this makes the changes more clear! Don't hesitate to let me know what additional questions come up.

maneesha committed:
update SOW links
